internal/resources/helmcharts/helm_charts_data_spurce_test.go

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Fix the filename. Check all other files for Helm.

Reply:
Updated PTAL